### DISCUSSION TITLE
UE 5.1 crashes when using sublevels( or start/end rooms)

### DISCUSSION THREAD

**Discussion by nooneuse** [shun126/UE5-DungeonGeneratorDemo#14] [Q&A]
I created a blank Level and set its Script Actor's parent class to the Dungeon Sublevel Script Actor, and then I put some static meshes and Player Start on it.
I registered it in: DungeonSublevelDatabase -> Generate Parameter.
Then when I regenerated the dungeon, Unreal Engine crashed

It shows:

_Assertion failed: false

UnrealEditor_DungeonGenerator! UDungeonSubLevelDatabase::ConvertImpl() [D:buildU5M-MarketplaceSyncLocalBuildsPluginTempHostProjectPluginsDungeonGeneratorSourceDungeonGeneratorPrivate SubLevelDungeonSubLevelDatabase.cpp:122]_

Unreal 5.1.1
DungeonGenerator 1.5.2/1.5.3/1.5.5 from marketplace

@shun126 says it's caused by a bug in 1.5.2, but I upgraded to 1.5.5 and the problem still exists.
I have a feeling it's because of the level streaming bShouldBlockOnLoad bug in ue 5.1: https://forums.unrealengine.com/t/level-streaming-flushlevelstreaming-causes-beginplay-not-to-be-called/1217419
https://forums.unrealengine.com/t/level-streaming-should-block-on-load-does-not-work/418904/2
This causes your level streaming object to not actually finish loading, and operations on this object may cause ue to crash, Please confirm this problem and help me figure out how to fix it

**Comment by shun126** [comment]
I am truly sorry for the inconvenience. I will investigate as soon as possible.
If possible, please update to version 1.5.10 or later.

**Comment by shun126** [comment]
Supported in plugin version 1.5.11
プラグインバージョン1.5.11で対応しました

#13
